front-end/src/App.tsx

```diff
@@ -36,16 +37,18 @@ function App() {
     }
   },[loggedInUser])
 
+  // TODO wrap the below in NodesContext as well, and then send it into pages and use it 
+  // in the Axios wrapper. OR make Nodes global.
```

Comment:
do you need this TODO since NodesContexts makes it global?

Reply:
good question, I'm not 100% familiar with how React Context works. I've updated the comment to not say to pass it around but to get in touch with Chris about how to do this.